service/build.go

```diff
@@ -161,7 +161,7 @@ func (uis *UIServer) modifyBuild(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	case "set_active":
-		err := model.SetBuildActivation(projCtx.Build.Id, putParams.Active, user.Id)
+		err := model.SetBuildActivation(projCtx.Build.Id, putParams.Active, user.Id, putParams.Abort)
```

Comment:
shouldn't this always be false?

Reply:
It should be what the parameters are. The set_active is a bit deceptive because this can both schedule or unschedule the build